docs/en/metadata_oidc_op.rst

```diff
@@ -129,10 +129,10 @@ The EC of an OP MUST configure a metadata of type **"openid_provider"**, that MU
     - See `OpenID.Discovery#OP_Metadata`_. See signature :ref:`supported_algs`.
     - |spid-icon| |cieid-icon|
   * - **request_object_encryption_alg_values_supported**
-    - See `OpenID.Discovery#OP_Metadata`_. See key encryption :ref:`supported_algs`.
+    - Until otherwise indicated by AgID, this MUST NOT be included.
     - |spid-icon|
   * - **request_object_encryption_enc_values_supported**
-    - See `OpenID.Discovery#OP_Metadata`_. See Content Encryption :ref:`supported_algs`.
+    - Until otherwise indicated by AgID, this MUST NOT be included.
```

Comment:
this is not CIE netrual

Reply:
@fmarino-ipzs vuoi dire qualcosa qui per cie?

Reply:
Eliminerei il riferimento esplicito alla AgID così da lasciare aperta la possibilità che a normarlo possa essere anche il Min Int per la CIE. Detto questo, non riesco a capire bene la ratio di questa frase. Inserite un parametro in una tabella che riporta i parametri del metadata OP, nella colonna "supported by" mettete SPID e poi dite che NON DEVE essere incluso??? Tanto vale non metterlo proprio. Quando AgID lo riterrà opportuno lo aggiungerà nella tabella e da quel momento gli OP dovranno prevederlo e supportarlo.

Reply:
"request_object_encryption_enc_values_supported" è previsto nelle LLGG OIDC SPID, quindi è necessario chiarire se e come deve essere usato, abbiamo ritenuto che sia una modifica formale, altrimenti bisognava modificare le LLGG con i relativi tempi di attesa.

Reply:
@AntonioFlorio , non è possibile fare, come già fatto altre volte in passato, non riportarlo qui e dare questa indicazione in un avviso che mi pare di capire comunque uscirà? Io sarei per avere qui una visione di quello che è e non di quello che NON è e che forse sarà. Ovviamente è una mia personale riflessione e non è bloccante per noi visto che è qualcosa di supportato (non ancora) solo da SPID.

Reply:
In ogni caso modificherei in "It MUST NOT be included, unless otherwise specified"

Reply:
mi sembra di capire che è supportato da LLGG SPID OIDC, pertanto la dicitura "until otherwise" risulterebbe in contrasto con le LLGG SPID OIDC

decidiamo il testo da adottare qui @AntonioFlorio

Reply:
Stiamo riportando in docsitalia quelle che abbiamo valutato come modifiche formali alle LLGG, per uniformare il più possibile SPID e CIE, sin da ora, senza dover attivare anche per queste cose l'iter ex art. 71 del CAD. 
Essendo, comunque, una modifica alle LLGG, è necessario esplicitarla e formalizzarla in maniera chiara, infatti la stessa indicazione è stata riportata anche nell'Avviso 41.
Per me va bene: "It MUST NOT be included, unless otherwise specified".

Reply:
> @AntonioFlorio , non è possibile fare, come già fatto altre volte in passato, non riportarlo qui e dare questa indicazione in un avviso che mi pare di capire comunque uscirà? Io sarei per avere qui una visione di quello che è e non di quello che NON è e che forse sarà. Ovviamente è una mia personale riflessione e non è bloccante per noi visto che è qualcosa di supportato (non ancora) solo da SPID.

Se in questo documento non è necessario specificare per gli sviluppatori le differenze rispetto alle LLGG possiamo anche eliminare il "request_object_encryption_enc_values_supported".

Reply:
Risolto da questo

https://github.com/italia/spid-cie-oidc-docs/pull/144/commits/cf6f9f5b127d9514c2daa95eb2f4e934f351e694